binding-service-impl/src/main/java/io/mosip/esignet/services/KeyBindingServiceImpl.java

```diff
@@ -69,18 +77,37 @@ public BindingOtpResponse sendBindingOtp(BindingOtpRequest bindingOtpRequest, Ma
 			log.error("Failed to send binding otp: {}", e);
 			throw new EsignetException(e.getErrorCode());
 		}
+		return getBindingOtpResponse(sendOtpResult);
+	}
+
+	@Override
+	public BindingOtpResponse sendBindingOtpV2(BindingOtpRequestV2 bindingOtpRequestV2, Map<String, String> requestHeaders)
+			throws EsignetException {
+		captchaHelper.validateCaptchaToken(bindingOtpRequestV2.getCaptchaToken(), "binding-otp");
+		log.debug("sendBindingOtp :: Request headers >> {}", requestHeaders);
+		SendOtpResult sendOtpResult;
+		try {
+			sendOtpResult = keyBindingWrapper.sendBindingOtp(bindingOtpRequestV2.getIndividualId(),
+					bindingOtpRequestV2.getOtpChannels(), requestHeaders);
+		} catch (SendOtpException e) {
+			log.error("Failed to send binding otp: {}", e);
+			throw new EsignetException(e.getErrorCode());
+		}
+		return getBindingOtpResponse(sendOtpResult);
```

Comment:
This complete code block is present in sendBindingOtp(BindingOtpRequest bindingOtpRequest, Map<String, String> requestHeaders). So instead of repeating the same code block can we simply invoke "sendBindingOtp" method.

Reply:
Changed and invoking sendBindingOtp(...) from V2 flow